playbooks/roles/cluster/tasks/workflow.yml

```diff
@@ -0,0 +1,146 @@
+    description: "{{ cluster_job_template_prerequisites_desc }}"
+    job_type: "{{ cluster_job_template_prerequisites_type }}"
+    playbook: "{{ cluster_job_template_prerequisites_playbook }}"
+    project: "{{ cluster_job_template_prerequisites_project }}"
+    credential: "{{ tower_credential_bundle_default_name }}"
+    state: present
+    inventory: "Provisioning_inventory"
+  register: cluster_create_prerequisites_out
+  until: cluster_create_prerequisites_out is succeeded
+  retries: 10
+  delay: 5
+
+- name: Associate additional credentials
+  shell: tower-cli job_template associate_credential --job-template "{{ cluster_job_template_prerequisites_name }}" --credential "{{ cluster_job_template_prerequisites_credentials }}"
+
+- name: Configure provision install job template
+  tower_job_template:
+    name: "{{ cluster_job_template_install_name }}"
+    description: "{{ cluster_job_template_install_desc }}"
+    job_type: "{{ cluster_job_template_install_type }}"
+    playbook: "{{ cluster_job_template_install_playbook }}"
+    project: "{{ cluster_job_template_install_project }}"
+    credential: "{{ tower_credential_bundle_default_name }}"
+    state: present
+    inventory: "Provisioning_inventory"
+  register: cluster_create_provision_install_out
+  until: cluster_create_provision_install_out is succeeded
+  retries: 10
+  delay: 5
+
+
+- name: Associate additional credentials
+  shell: tower-cli job_template associate_credential --job-template "{{ cluster_job_template_install_name }}" --credential "{{ cluster_credential_bundle_aws_name }}"
+
+- name: Associate additional credentials
+  shell: tower-cli job_template associate_credential --job-template "{{ cluster_job_template_install_name }}" --credential "{{ cluster_job_template_install_credentials }}"
+
+- name: Configure post-install job template
+  tower_job_template:
+    name: "{{ cluster_job_template_post_install_name }}"
+    description: "{{ cluster_job_template_post_install_desc }}"
+    job_type: "{{ cluster_job_template_post_install_type }}"
+    playbook: "{{ cluster_job_template_post_install_playbook }}"
+    project: "{{ cluster_job_template_post_install_project }}"
+    credential: "{{ tower_credential_bundle_default_name }}"
+    state: present
+    inventory: "Provisioning_inventory"
+  register: cluster_create_post_install_out
+  until: cluster_create_post_install_out is succeeded
+  retries: 10
+  delay: 5
+
+- name: "Create workflow: {{ cluster_workflow_job_template_name }}"
+  tower_workflow_template:
+    name:  "{{ cluster_workflow_job_template_name }}"
+    description: "{{ cluster_workflow_job_template_desc }}"
+    state: present
+    organization: "{{ tower_organization }}"
+
+- name: Create workflow schema
+  template:
+    src: workflow_schema.yml.j2
+    dest: "cluster_workflow_schema.yml"
+
+- name: Retrieve list of AWS Credential Bundles
+  shell: "tower-cli credential list --credential-type {{ aws_cred_type_id.stdout }} -f json"
```

Comment:
`aws_cred_type` var is not being set, think you need to add this code block also: https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/tasks/workflow.yml#L35-L37

Reply:
Sure, added that block of code in.